service/src/main/java/bio/terra/workspace/db/WorkspaceDao.java

```diff
@@ -165,17 +181,23 @@ public Workspace getWorkspace(UUID id) {
   @WriteTransaction
   public boolean updateWorkspace(
       UUID workspaceId,
+      @Nullable String userFacingId,
       @Nullable String name,
       @Nullable String description,
       @Nullable Map<String, String> propertyMap) {
-    if (name == null && description == null && propertyMap == null) {
+    if (userFacingId == null && name == null && description == null && propertyMap == null) {
       throw new MissingRequiredFieldException(
-          "Must specify name, description, or properties to update.");
+              // "ID" instead of "userFacingId" because end user sees this.
+              "Must specify ID, name, description, or properties to update.");
```

Comment:
suggestion: just "No update provided."

Reply:
Done, changed to `"Must specify field to update."`